enterprise/server/crypter_service/crypter_service.go

```diff
@@ -864,10 +864,6 @@ func (c *Crypter) enableEncryption(ctx context.Context, kmsConfig *enpb.KMSConfi
 		return err
 	}
 
-	if !c.env.GetCache().SupportsEncryption(ctx) {
```

Comment:
Are you going to clean this up in a future PR?

Reply:
I am thinking of renaming it and re-using it to display whether a customer has a dedicated partition when performing impersonation.